tests/db_integrity/test_data_updates.py

```diff
@@ -261,14 +252,9 @@ def test_obs_data_read_header_diff():
      - BSD GCMD different data added - header changed so hash will be different but data will be the same
     Expect that GCMD (and CRDS) data can still be accessed.
     """
-    clear_test_stores()
-    # Load BSD data - CRDS data
     bsd_data_read_crds()
```

Comment:
Please add the clarifying comments be added back here.

Reply:
Done